snappy/binaries.go

```diff
@@ -71,29 +71,31 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 	var templateOut bytes.Buffer
 	t := template.Must(template.New("wrapper").Parse(wrapperTemplate))
 	wrapperData := struct {
-		SnapName    string
-		AppName     string
-		SnapArch    string
-		SnapPath    string
-		Version     string
-		Revision    int
-		UdevAppName string
-		Home        string
-		Target      string
-		AaProfile   string
-		OldAppVars  string
-		NewAppVars  string
+		SnapName     string
+		AppName      string
+		SnapArch     string
+		SnapBasePath string
+		SnapPath     string
+		Version      string
+		Revision     int
+		UdevAppName  string
+		Home         string
+		Target       string
+		AaProfile    string
+		OldAppVars   string
+		NewAppVars   string
 	}{
-		SnapName:    app.Snap.Name(),
-		AppName:     app.Name,
-		SnapArch:    arch.UbuntuArchitecture(),
-		SnapPath:    pkgPath,
-		Version:     app.Snap.Version,
-		Revision:    app.Snap.Revision,
-		UdevAppName: app.SecurityTag(),
-		Home:        "$HOME",
-		Target:      actualBinPath,
-		AaProfile:   app.SecurityTag(),
+		SnapName:     app.Snap.Name(),
```

Comment:
+1, but ditto my comment about `stripGlobalRootDir` and the fact that this will add a lot of diff noise. Do you want me to do that here, or in another PR?